### ISSUE TITLE
ui: One graph missing on console

### ISSUE THREAD

**Issue by bdarnell** [cockroachdb/cockroach#8092]
On [gamma](https://gamma.gce.cockroachdb.com:8080/) the "Reads" graph is missing but everything else seems to be loading OK. Nothing is logged in the javascript console.

**Comment by petermattis** [comment]
@maxlang, @mrtracy It's not clear if this problem is ephemeral, so take a look as soon as you can.

**Comment by mrtracy** [comment]
This one is bizarre.  No errors in the logs, and a KV scan suggest that all of the time series keys are present and accounted for.  Trying to analyze the network requests/responses now.

**Comment by mrtracy** [comment]
It has data until 1:30pm the previous night; after that it can't query the data.  Which is odd, because the data definitely appears in the k/v store.  

Could this be an issue with the inconsistent reads?  Is there a 12-hours out of date replica answering these queries?

**Comment by bdarnell** [comment]
I checked all the IPs in the gamma cluster directly. The graph is present on two (https://104.196.166.97:8080/ for example) and missing on the other four. So it does look like a case of inconsistent reads being very inconsistent.

**Comment by mrtracy** [comment]
Okay, I checked the RangeID for the range that contains this data, and lo and behold the range does have a consistency error:

From https://gamma.gce.cockroachdb.com:8080/#/raft/ranges?_k=4dbif6
![screen shot 2016-07-28 at 4 37 16 pm](https://cloud.githubusercontent.com/assets/6969858/17228614/b7119c68-54e1-11e6-9237-244463073290.png)

Verified that is the range via Meta2 keys.  That seems wildly inconsistent, why do the other ranges agree with it?

**Comment by bdarnell** [comment]
OK, so that replica was removed from the range, not GC'd, and then re-added. And for some reason it's not getting updated from the other range. I'll file a new issue for the core part of this bug.

For this one, I think we should change the way the UI does TS queries, since an inconsistent read can be inconsistent by an unbounded amount. I'd prefer to just do consistent reads, since the level of availability that allows inconsistent reads to be served but not consistent ones is narrow. (and if you can't serve consistent reads then new data can't be written either). If we still want to be able to show *something* when consistent reads are unavailable, then we should probably try a consistent read first and do an inconsistent read only after that has failed.
